src/internal/CairoFrame.ts

```diff
@@ -0,0 +1,181 @@
+import { CamOverlayDrawingAPI, Align } from '../CamOverlayDrawingAPI';
```

Comment:
CairoFrame potrebujes jako uzivatel volat a nemel by byt v internal.

Reply:
ResourceManager pak to same, protoze se predava v konstruktoru CairoFrame

Reply:
!